QUESTIONS.md

```diff
@@ -3,3 +3,8 @@
 What questions can you come up with?
 
 ## Cohort Questions
+question1:
+Most web dev job posting required candidates to have a bachelor in CS. what about those of us who don not have a CS bachelor?
```

Comment:
Don't worry about it, apply anyway. This course is meant to make up for that difference in experience.